src/middlewares.js

```diff
@@ -0,0 +1,52 @@
+const notFound = (req, res, next) => {
+  res.status(404);
+  const error = new Error(`🔍 - Not Found - ${req.originalUrl}`);
+  next(error);
+};
+
+// const errorHandler = (err, req, res) => {
```

Comment:
Avoid sending commented lines to PR.

Reply:
i need to work on this handler, so commented, all unnecessary comments will be removed for pr's